client/src/app/components/identity/IdentityStardustResolver.tsx

```diff
@@ -0,0 +1,256 @@
+import { IdentityStardustResolverState } from "./IdentityStardustResolverState";
+
+class IdentityStardustResolver extends AsyncComponent<
+    RouteComponentProps<IdentityStardustResolverProps> & { protocolVersion: string },
+    IdentityStardustResolverState
+> {
+    /**
+     * Timer to check to state update.
+     */
+    private _timerId?: NodeJS.Timer;
+
+    /**
+     * placeholder when messageId is not available.
+     */
+    private readonly EMPTY_MESSAGE_ID = "0".repeat(64);
+
+    constructor(props: RouteComponentProps<IdentityStardustResolverProps> & { protocolVersion: string }) {
+        super(props);
+
+
+        this.state = {
+            did: props.match.params.did,
+            aliasId: getAliasId(props.match.params.did ?? ""),
+            error: false,
+            errorMessage: ""
+        };
+    }
+
+    public async componentDidMount(): Promise<void> {
+        super.componentDidMount();
+
+        if (!this.state.did) {
+            return;
+        }
+
+        const res = await ServiceFactory.get<IdentityService>("identity").resolveIdentityStardust(
+            this.state.did,
+            this.props.match.params.network
+        );
+
+        if (res.error) {
+            this.setState({
+                error: true,
+                errorMessage: res.error
+            });
+            return;
+        }
+
+        this.setState({
+            resolvedIdentity: res
+        });
+    }
+
+    /**
+     * The component will unmount so update flag.
+     */
+    public componentWillUnmount(): void {
+        super.componentWillUnmount();
+        if (this._timerId) {
+            clearTimeout(this._timerId);
+            this._timerId = undefined;
+        }
+        IdentityDiffStorageService.instance.clearAll();
+    }
+
+
+    /**
+     * Render the component.
+     * @returns The node to render.
+     */
+    public render(): ReactNode {
+        return (
+            <div>
+                <div className="row space-between wrap">
+                    <div className="row middle">
+                        <h1>
+                            Decentralized Identifier
+                        </h1>
+                        <Modal icon="info" data={welcomeMessage} />
+                    </div>
+                </div>
+                <div className="section">
+                    <div className="section--data">
+                        <div className="label">DID</div>
+                        <div className="row middle value code highlight margin-b-s">
+                            <div className="margin-r-t">{this.props.match.params.did}</div>
+                            <CopyButton copy={this.props.match.params.did} />
+                        </div>
+
+                        {/* ALIAS ID */}
+                        {this.state.resolvedIdentity?.document &&
+                            !this.state.error && (
+                                <Fragment>
+                                    {/* ALIAS ID */}
```

Comment:
I don't really see the value in these comments in the JSX, what we are looking at is already visible from the label.